.github/workflows/test.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
     env:
       PANTS_REMOTE_CACHE_READ: 'false'
       PANTS_REMOTE_CACHE_WRITE: 'false'
-    if: (${{ github.repository_owner == 'pantsbuild' }}) && needs.docs_only_check.outputs.docs_only
-      == 0
+    if: (github.repository_owner == 'pantsbuild') && (needs.docs_only_check.outputs.docs_only
+      != 'DOCS_ONLY')
```

Comment:
With this new change, we only set the output at all if this is in fact a docs-only change. In other cases we get an empty string. So rather than setting it to "1" or "true", or something that would imply that "0" or "false" are also possible values, we set it to some sentinel string, and we check the inverse with `!= <the value>` instead of `== <the inverse value>`